src/h3lib/include/h3api.h

```diff
@@ -275,6 +275,14 @@ double H3_EXPORT(edgeLengthM)(int res);
 int64_t H3_EXPORT(numHexagons)(int res);
 /** @} */
 
+/** @defgroup getBaseCells getBaseCells
+ * Functions for getBaseCells
+ * @{
+ */
+/** @brief provides all base cells */
+void H3_EXPORT(getBaseCells)(H3Index *out);
```

Comment:
I am wondering about the name of this function, since in the library a "base cell" usually refers to an integer identifier 0-122. Maybe getRes0Indexes or something?